website/server/models/group.js

```diff
@@ -9,32 +9,23 @@ import { // eslint-disable-line import/no-cycle
 } from './user';
 import shared from '../../common';
 import { model as Challenge } from './challenge'; // eslint-disable-line import/no-cycle
-import {
-  chatModel as Chat,
-  setUserStyles,
-  messageDefaults,
-} from './message';
+import { chatModel as Chat, setUserStyles, messageDefaults } from './message';
 import * as Tasks from './task';
 import { removeFromArray } from '../libs/collectionManipulators';
 import payments from '../libs/payments/payments'; // eslint-disable-line import/no-cycle
 import { // eslint-disable-line import/no-cycle
   groupChatReceivedWebhook,
   questActivityWebhook,
 } from '../libs/webhook';
-import {
-  InternalServerError,
-  BadRequest,
-  NotAuthorized,
-} from '../libs/errors';
+import { InternalServerError, BadRequest, NotAuthorized } from '../libs/errors';
 import baseModel from '../libs/baseModel';
 import { sendTxn as sendTxnEmail } from '../libs/email'; // eslint-disable-line import/no-cycle
 import { sendNotification as sendPushNotification } from '../libs/pushNotifications'; // eslint-disable-line import/no-cycle
-import { // eslint-disable-line import/no-cycle
+import {
+  // eslint-disable-line import/no-cycle
```

Comment:
does this actually work? o.O I think it's more readable if we keep it on the same line as the `import` at least...

Reply:
reverted back to `develop`